README.md

```diff
@@ -23,6 +38,23 @@ jobs:
       matrix:
         node-versions: ['10.16.3', '14.9']
     runs-on: self-hosted
+  # Cache service for GHE / self-hosted runners
+    services:
```

Comment:
hmm this is ok but self-hosted setup mostly will need extra setup.

When self-hosted is running with docker in docker

```
Error: Container feature is not supported when runner is already running inside container.
```

When self-hosted is running on a Linux

```
Error: docker: command not found
```

Reply:
This is good as an optional feature. I worry that it over complicates a simple http server.

Reply:
I have merged the changes. Thank you.